doc/api/cli.md

```diff
@@ -51,6 +51,28 @@ entry point, the `node` command will accept as input only files with `.js`,
 [`--experimental-wasm-modules`][] is enabled; and with no extension when
 [`--experimental-default-type=module`][] is passed.
 
+## Sub-commands
+
+### `node run [command]`
```

Comment:
This syntax should include the arguments, something like: `node run [command] [arguments]`?